.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6" # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+script:
+  - python setup.py sdist bdist_wheel
+  - pip install dist/wavefront_cli-*.whl
```

Comment:
I'd use `python setup.py install` instead of building the wheel and using pip locally

Reply:
I have updated the script to not build the wheel file and install it from the wheel.
Now the script will run below command to install the wavefront-cli

- pip install .

Below command also works fine

- python setup.py develop

I haven't use the python setup.py install as it is not creating the wave file under bin directory.